### PR TITLE
docs: Heading component documentation re-organization

### PR DESCRIPTION
**Pull request by osbornedesign**
image

### CODE REVIEW

**Comment by osbornedesign** [comment]
@nathanyoung can you check the Visual Regression Tests and make sure this is how we want to display them. I wasn't sure how to go about breaking them into sections but then was questioning if that is even necessary

**Comment by nathanyoung** [comment]
> @nathanyoung can you check the Visual Regression Tests and make sure this is how we want to display them. I wasn't sure how to go about breaking them into sections but then was questioning if that is even necessary

It looks like you've got 1 too many loops and the visual regression test story. There are so many loops that chromatic isn't even rendering it because it's too long.

**Comment by nathanyoung** [comment]
Screen Shot 2021-02-12 at 12 50 37 PM

I think we want a story for
- each size
- each color
- each size and color combination
- responsive size

**Comment by osbornedesign** [comment]
> > @nathanyoung can you check the Visual Regression Tests and make sure this is how we want to display them. I wasn't sure how to go about breaking them into sections but then was questioning if that is even necessary
> 
> It looks like you've got 1 too many loops and the visual regression test story. There are so many loops that chromatic isn't even rendering it because it's too long.

It looks like there are too many loops but it's because of the "level" prop. It is showing each size as "h1, h2, h3..."
image

File: src/components/Heading/Heading.Overview.stories.mdx [hunk 1]
```diff
@@ -71,7 +74,7 @@ The variant of the heading is inherited from its parent, but can be set to brand
 <Canvas>
   <Story name="Colors">
```
Comment (nathanyoung) [inline src/components/Heading/Heading.Overview.stories.mdx:75]:
```suggestion
  <Story name="Variant">
```

File: src/components/Heading/Heading.VisualTests.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    ))}
+  </Box>
+);
+
+export const Size = SizeTemplate.bind({});
+Size.args = {
+  children: 'Hello world!',
+};
+
+export const Color = ColorTemplate.bind({});
```
Comment (nathanyoung) [inline src/components/Heading/Heading.VisualTests.stories.tsx:45]:
```suggestion
export const Variant = ColorTemplate.bind({});
```

**Review by nathanyoung — APPROVED** [review]

**Pull request merged**